### ISSUE TITLE
`sync_aggregate` vs `sync_committee` name

### ISSUE THREAD

**Issue by protolambda** [ethereum/eth2.0-specs#2433]
- the operation in the beacon block is called `sync_aggregate`
- the processing function is called `process_sync_committee`
- the processing argument is called `sync_aggregate`, the type is `SyncAggregate`
- the py tests module is called `test_process_sync_committee`
- the ssz file in the test vector output is called `sync_aggregate.ssz_snappy`
- the test handler is named `sync_committee`
- the test format doc of the `operations` runner is wrong: the table says the operation name is `sync_aggregate`, while it's really `sync_committee` in the test dir structure. The input name is indeed `sync_aggregate` though.

Can we make this consistent, and just pick one or the other in all places? I like `sync_aggregate` more: since there may be many other sync committee related things, but the `SyncAggregate` is the exact type we are dealing with in block processing.

**Comment by hwwhww** [comment]
To me:
- Variable `sync_aggregate` is a `SyncAggregate` object
- Variable `sync_committee` is a list of the validator indices.
- Variable `aggregate` *usually* is an aggregate signature. But sometimes it's not.

---

It would nice to also rename all the `SyncAggregate` instances to `sync_aggregate`. For example:
```python
process_sync_committee(state: BeaconState, aggregate: SyncAggregate)
```

->

```python
process_sync_aggregate(state: BeaconState, sync_aggregate: SyncAggregate)
```

**Comment by djrtwo** [comment]
agreed with @hwwhww's description. And for `process_` function names, we do `process_THE_THING` not `process_THE_GROUP_THAT_DID_THE_THING`, so `process_sync_aggregate` makes sense to me.

**Comment by protolambda** [comment]
Fixed in #2469
